src/main/java/it/gov/pagopa/pu/classification/service/assessments/AssessmentsServiceImpl.java

```diff
@@ -24,6 +25,8 @@
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
 import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 @Lazy
```

Comment:
I missed this @Lazy annotation, can we remove it?
@Lazy should be used only in activities repo